cincinnati/src/plugins/internal/graph_builder/release_scrape_dockerv2/registry/mod.rs

```diff
@@ -152,10 +157,16 @@ impl Registry {
         bail!("unsupported registry format {}", src)
```

Comment:
This is somehow failing in CI, with [this run][1] failing on:

```
+ oc -n openshift-update-service wait --for=condition=Ready pod -l app=e2e
timed out waiting for the condition on pods/e2e-6ddff6cf68-sfln9
timed out waiting for the condition on pods/e2e-857df64b8b-46rwh
timed out waiting for the condition on pods/e2e-857df64b8b-k8gff
+ failed=1
+ [[ 1 -eq 0 ]]
+ attempt=61
```

From [gathered pod logs][2]:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_cincinnati/895/pull-ci-openshift-cincinnati-master-e2e/1765431628563222528/artifacts/e2e/gather-extra/artifacts/pods/openshift-update-service_e2e-6ddff6cf68-sfln9_graph-builder.log | tail
                remove_consumed_metadata: false,
                include_conditional_edges: true,
            },
        ],
        tracing_endpoint: None,
    }
Error: trying to extract Registry from quay.io/openshift-release-dev/ocp-release

Caused by:
    unsupported registry format quay.io/openshift-release-dev/ocp-release
```

I'm not entirely sure what's going on there, but seems likely to have been broken by this pull, since it's about credential-lookup.

[1]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_cincinnati/895/pull-ci-openshift-cincinnati-master-e2e/1765431628563222528#1:build-log.txt%3A908-914
[2]: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_cincinnati/895/pull-ci-openshift-cincinnati-master-e2e/1765431628563222528/artifacts/e2e/gather-extra/artifacts/pods/